### ISSUE TITLE
[6.4.9] Unknown exception created by Scroller

### ISSUE THREAD

**Issue by dvilchez-qatium** [storybookjs/storybook#17161]
**Describe the bug**
I was upgrading my storybook project from version 5.3.21 to 6.4.9. After the upgrade starting the app fails with this error:

![2022-01-07 at 11 16](https://user-images.githubusercontent.com/92300685/148529262-4e265299-055a-44b7-8f36-7da3f1bd4387.png)

**To Reproduce**
Update storybook in a project using react 17.0.2 (I have tried to create a reproduction but the script hasn't worked for me)

**System**
Environment Info:

  System:
    OS: macOS 12.1
    CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
  Binaries:
    Node: 16.11.1 - ~/.nvm/versions/node/v16.11.1/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v16.11.1/bin/yarn
    npm: 8.0.0 - ~/.nvm/versions/node/v16.11.1/bin/npm
  Browsers:
    Chrome: 97.0.4692.71
    Safari: 15.2
  npmPackages:
    @storybook/addon-a11y: ^6.4.9 => 6.4.9
    @storybook/addon-actions: ^6.4.9 => 6.4.9
    @storybook/addon-docs: ^6.4.9 => 6.4.9
    @storybook/addon-knobs: ^5.3.21 => 5.3.21
    @storybook/addon-viewport: ^6.4.9 => 6.4.9
    @storybook/preset-typescript: ^3.0.0 => 3.0.0
    @storybook/react: ^6.4.9 => 6.4.9

**Additional context**
It works if I downgrade the project to react 16.14.0

**Comment by joepuzzo** [comment]
I am also getting exact same error when updating to react 17 with storybook

**Comment by lurch83** [comment]
What's the workaround that's marked? Downgrade to react 16.14?

**Comment by mohamedMok** [comment]
Hello, 
We had the same issue on our storybook (for vue3) using lerna.
We specified in our main `package.json` the `16.14` version

In one of our packages, a dependency was using react `17` and when lerna detected, it was the version used.

@lurch83 You're right, to fix it we had to force the react version (16.14)  in our storybook package. 

@shilman I would love to help or contribute to avoiding this type of workaround with Storybook.
